sklearn_crfsuite/_fileresource.py

```diff
@@ -35,6 +35,7 @@ def cleanup(self):
         if self.name is not None:
             try:
                 os.unlink(self.name)
+                os.close(self.fd)
```

Comment:
Not a particular reason. Yeah maybe it is "nicer" that the file descriptor is closed before, but it makes no difference in this context for these reasons (but correct me if I am wrong):
- unlink does not modify (close) the file descriptor we close the instruction after and,
- since the library is single threaded (is it right?) the file descriptor will not be used by anyone else (to access a file that is no more there) than the close instruction. Closing a file decriptor of a file that doesn not exists anymore it is not an error, it just removes a variable in the process control block of the process :)

buy yeah, change it if you want :)